src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/Create_AddJsonResponseType_AsMetadata.generated.txt

```diff
@@ -0,0 +1,374 @@
+//------------------------------------------------------------------------------
```

Comment:
Are all these new baseline tests necessary? The original intention of the baseline tests were to only have a handful of tests baselined that covered a wide swath of general features. Having a lot of baseline tests adds to the effort when the code needs to be updated.

Reply:
I've reduced the number I create and instead just selected baselines for a scenarios that I want to make sure we cover at least once.